src/battle/battle_icon.c

```diff
@@ -72,37 +71,37 @@ __attribute__((aligned(4))) static const u8 BattleIcon_PaletteIndexMap[] = {
 };
 
 /*
-* Maps move types (Physical, Special, Status) to the corresponding NARC member in pl_batt_obj.narc containing the type icon.
-*/
+ * Maps move types (Physical, Special, Status) to the corresponding NARC member in pl_batt_obj.narc containing the type icon.
+ */
 __attribute__((aligned(4))) static const u32 BattleIcon_MoveType_NARCMemberMap[] = {
```

Comment:
**nomenclature**: We call this "move class" to explicitly delineate it from types.